joomla/joomla-cms
# [5.0] Drop tinyMCE own languages

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/40408 .

### Summary of Changes

Use the official translation files

### Testing Instructions

Run `npm ci` and check that changing the language in tinyMCE still works as before

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Link to documentations
Please select:



@tecpromotion @brianteeman


A note here: this could be alternative done as a GitHub action similar to the Joomla languages with the benefit that there will not be an additional nom dependency for the `adm-zip` module and also keep the tools execution time shorter...
